src/lib/components/Collapsible/root.svelte

```diff
@@ -1,7 +1,17 @@
 <script lang="ts" context="module">
 	export type CollapsibleRootProps = BaseProps & {
+		/**
+		 * The controlled open state of the collapsible.
+		 */
 		open?: boolean;
+		/**
+		 * The open state of the collapsible when it is initially rendered.
+		 * Use when you do not need to control its open state.
+		 */
 		defaultOpen?: boolean;
+		/**
+		 * When `true`, prevents the user from interacting with the collapsible.
+		 */
```

Comment:
Nice! Does this appear in autocomplete perhaps?

Reply:
Yes, when you hover the property when using the component, it writes the docs out in vscode. Would be cool if there was a way to extract these comments for automatic docs generation 🙄 (i was trying last night, but didn't find an obvious way to do so without getting too deep in typescript compiler code)

Reply:
Ooh, that would be amazing!